docs/guides/backends/s3/index.md

```diff
@@ -61,65 +56,82 @@ secret/minio-secret created
 
 {{< notice type="warning" message="If you are using a Minio backend, make sure that you are using `AWS_ACCESS_KEY_ID` instead of `MINIO_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` instead of `MINIO_SECRET_KEY`." >}}
```

Comment:
Check if we need this one.

Reply:
Yes, Needed. This information indicates to alert users that they should use AWS_ACCESS_KEY_ID instead of MINIO_ACCESS_KEY.